singularity-eos/eos/eos_eospac.hpp

```diff
@@ -1274,43 +1270,52 @@ inline std::size_t EOSPAC::SetDynamicMemory(char *src, const SharedMemSettings &
 }
 inline std::size_t EOSPAC::SharedMemorySizeInBytes() const { return shared_size_; }
 
-SG_PIF_NOWARN
 template <typename Indexer_t>
 PORTABLE_INLINE_FUNCTION Real EOSPAC::TemperatureFromDensityInternalEnergy(
     const Real rho, const Real sie, Indexer_t &&lambda) const {
+#if SINGULARITY_ON_DEVICE
+  EOS_ERROR("EOSPAC calls not supported on device\n");
```

Comment:
I forget, but is there a reason to prefer `EOS_ERROR` or `PORTABLE_ERROR`?

Reply:
Done.